src/components/Navbar.js

```diff
@@ -58,17 +71,23 @@ const Navbar = ({ toggleDarkMode, mode }) => {
             </>
           )}
           <Button color="inherit" component={Link} to="/home">
-            Home
+            <MusicVideoIcon/>
+          </Button>
```

Comment:
Looks great! 

One thing we should do for accessibility and making it clear to all users what this icon does, is add an `aria-label` (this will be read to screenreaders, and will show if the icon fails to load) and wrap this in a [Tooltip](https://mui.com/material-ui/react-tooltip/) component so that a tooltip shows when you hover over the button. We should do this for all our navbar buttons that are just icons now with no text.

```suggestion
          <Tooltip title="Home" arrow placement="bottom">
            <IconButton color="inherit" component={Link} to="/home" aria-label="link to home page">
              <MusicVideoIcon />
            </IconButton>
          </Tooltip>
```

Reply:
In my suggestion above I also switched the use of the `Button` component for an `IconButton` component, but either is really fine here.

Reply:
done!